ui/app/css/lib.css

```diff
@@ -196,6 +196,23 @@ hr.horizontal-line {
   align-items: center;
   justify-content: center;
   padding: 4px;
+  z-index: 1;
+}
+
+.keyring-label {
```

Comment:
Yeah, I wish I'd known that.  If we ever decide to actually adopt that pattern, we should have fat comments explaining the intention of each file, so these patterns aren't passed only by word of mouth.